django/utils/choices.py

```diff
@@ -20,6 +19,8 @@ def __iter__(self):
 
 def normalize_choices(value, *, depth=0):
     """Normalize choices values consistently for fields and widgets."""
+    # Avoid circular import (#34807)
```

Comment:
I'd suggest something like "# Avoid circular import when importing django.forms."  (Unless it's impractical, I'd avoid referencing a ticket rather than explaining the issue inline.)

Reply:
Thanks, applying.